tensorflow_datasets/image/vis_dial.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Visual Dialog Dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import os
+import re
+import fnmatch
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """\
+@inproceedings{visdial,
+  title={{V}isual {D}ialog},
+  author={Abhishek Das and Satwik Kottur and Khushi Gupta and Avi Singh
+    and Deshraj Yadav and Jos\'e M.F. Moura and Devi Parikh and Dhruv Batra},
+  booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
+  year={2017}
+}
+"""
+
+_Train_Dialogs = "https://dl.dropboxusercontent.com/s/ix8keeudqrd8hn8/visdial_1.0_train.zip"
```

Comment:
Constants should be capitalized for consistency. You can use pylint to help you with code formatting https://www.pylint.org/